ansible/idr-omero.yml

```diff
@@ -55,4 +55,31 @@
   roles:
   - role: openmicroscopy.omero-server
 
+  tasks:
+    - name: Override Bio-Formats artifacts in lib/server
+      become: yes
+      become_user: "{{ omero_server_system_user }}"
+      get_url:
+        url: "{{ idr_bf_baseurl }}/{{ item }}/{{ idr_bf_release }}/{{ item }}-{{ idr_bf_release }}.jar"
+        dest: "{{ omero_server_basedir }}/{{ omero_server_symlink }}/lib/server/{{ item }}.jar"
+        force: yes
+      with_items: "{{ idr_bf_components }}"
+      notify: omero-server restart omero-server
+    - name: Override Bio-Formats artifacts in lib/client
```

Comment:
`Override Bio-Formats artifacts in lib/client` is missing from `idr-omero-readonly.yml`.

If you're concerned about duplicate code one option is to move it into another playbook that's called after: https://github.com/IDR/deployment/blob/IDR-0.6.8-1/ansible/idr-01-install-idr.yml#L15

Something like
```yaml
roles:
  - ome.omero_common
tasks:
  - name: Override Bio-Formats artifacts in lib/server
    notify: restart omero-server # Generic restart handler from ome.omero_common instead of the omero-server internal one
  - name: Override Bio-Formats artifacts in lib/client
    notify: restart omero-server
  - name: Remove OMERO scripts
    notify: restart omero-server
```

Reply:
Thanks for the suggestion. I initially skipped `lib/client` from `idr-omero-readonly.yml` as we would not run a `bin/omero import` in these servers. I like the idea of migrating the custom IDR step into a separate playbook, both for compartmentalization and ease of testing of future IDR/Bio-Formats versions. Pushed in 3678c87

Reply:
Ah, from https://travis-ci.org/IDR/deployment/jobs/565055434, the downside of using `omero_common` is the loss of knowledge about the OMERO.server system user

````
    fatal: [idr-omeroreadonly-docker]: FAILED! => {"msg": "The field 'become_user' has an invalid value, which includes an undefined variable. The error was: 'omero_server_system_user' is undefined"}
```

Reply:
Get rid of `become_user`? It should be fine to run the copy as `root`, there's no reason it needs to be writeable by OMERO (long term it'd be nice if most of OMERO.server was owned by root but that's a separate discussion).

Reply:
I think it's also fine to hard-code the server path `/opt/omero/server/OMERO.server` or `{{ omero_common_basedir }}/server/OMERO.server` instead of depending on the omero-server role variables, e.g. as in https://github.com/IDR/deployment/blob/IDR-0.6.8-1/ansible/management-prometheus.yml#L46